### PR TITLE
Replace vpc vpn code with vpn module

### LINKED ISSUE #396: Use VPN module
Repace vpc code with the vpn module

### PR DESCRIPTION
**Pull request by limed**
Fixes #396

### CODE REVIEW

File: modules/vpc/main.tf [hunk 1]
```diff
@@ -1049,86 +1047,21 @@ module "user_management" {
...
-  }
-
-  route_table_id = "${element(aws_route_table.private.*.id, count.index)}"
+  enabled           = "${var.enabled * var.enable_vpn}"
+  region            = "${var.aws_region}"
+  arenas            = "${var.arenas}"
+  account_name      = "${var.account_name}"
+  technical_contact = "${var.technical_contact}"
 
-  destination_cidr_block = "10.0.0.0/8"
```
Comment (gozer) [inline modules/vpc/main.tf]: what hapenned to destination_cidr_block ?

File: input.tf [hunk 2]
```diff
@@ -146,6 +146,7 @@ variable vpn {
     ipsec_target  = "63.245.214.100"
     ipsec_network = "10.0.0.0/8"
     bgp_asn       = "65022"
+    output_config = false
```
Comment (gozer) [inline input.tf]: Just so I understand, right now, we just don't generate the config file for the firewall then, right ?

File: modules/vpc/main.tf (hunk 1 above)
Comment (limed) [reply modules/vpc/main.tf]: Good catch, I missed this and will add it back again

File: input.tf (hunk 2 above)
Comment (limed) [reply input.tf]: yes we don't this is optional and basically what the current status quo is

**Pull request merged**